s3/pkg/helper/file.go

```diff
@@ -0,0 +1,14 @@
+package helper
+
+import (
+	"os"
+)
+
+func FileExists(path string) bool {
```

Comment:
this name and thsi function is imho, incomplete...What happens is the file exists and the mode is NOT regular ?